src/vcount.c

```diff
@@ -0,0 +1,34 @@
+#include "global.h"
+#include "vcount.h"
+
+void sub_08006E44(void)
+{
+    u16 idx = gUnk_03003230;
+    u16 vc = REG_VCOUNT;
+    struct Unk_030031D0 *ptr2 = gUnk_030031D0;
+    struct Unk_030031D0 *ptr = &ptr2[idx];
+    if (ptr->unk4 == vc)
+    {
+        *gUnk_030031D0[idx].unk0 = ptr->unk6;
+        REG_DISPSTAT &= 0xff;
+        REG_DISPSTAT = ptr->unkC * 0x100 | REG_DISPSTAT;
```

Comment:
Here `<< 8` is more reasonable than `* 0x100`. Same below.

And does `|=` work here?